docs/source/release/v6.0.0/mantidworkbench.rst

```diff
@@ -9,6 +9,7 @@ Improvements
 ############
 
 - Enable workbench to plot 1D spectrum from an IMDHistoWorkspace whose non-integrated dimension is equal to 1 but not to launch slice viewer.
+- Enabled the plotting of individual functions in the Workbench fit browser. This replicates a feature that was in MantidPlot
```

Comment:
Would it be good to mention how to us this new feature (i.e. by Right-clicking a function in the browser and selecting `Plot`)?